engines/csvengine.py

```diff
@@ -100,6 +97,11 @@ def table_exists(self, dbname, tablename):
         tablename = self.table_name(name=tablename, dbname=dbname)
         return os.path.exists(tablename)
 
+    def to_csv(self):
+        """Export sorted version of CSV file"""
+        from retriever.lib.tools import sort_csv
```

Comment:
Is there a reason for this import to be inside the function instead of at the top of the module? If not let's move it up.